### DISCUSSION TITLE
Moving Volumes between hosts.

### DISCUSSION THREAD

**Discussion by afro-coder** [containers/podman#12105] [Q&A] [answered]
Hey,

I was wondering how do you all move volumes between hosts if needed.
Do you do a SCP and then re-run the containers or is there some other way.

Also how does one use `podman system connection` like if I wanted to run the container on a remote host

Thanks

**Comment by rhatdan** [comment, marked as answer]
podman volume export and podman volume import. Can create volumes out of the contents of a volume and create the volume on a new host. Or use network based file system.

Setup podman to run on a remote server which you have ssh access to, and then create a connection using podman system connection create

**Reply by afro-coder** [reply]
Oh, I somehow never seen that, my bad. Thank you!

For the system connection, I created a connection as such
```
podman system connection list
Name        Identity         URI
prod*       .ssh/sshkey  ssh://leon@remotehost/run/user/1001/podman/podman.sock
```
And to use this I would have to do this I'm guessing
```
podman --remote pull alpine
```
Since I see it getting logged on the remote server.

Thank you once again!
